setup.py

```diff
@@ -5,19 +5,17 @@
 import sys
 import setuptools
 import subprocess
+from warnings import warn
 
 
-ERFA_SRC = os.path.join('liberfa', 'erfa', 'src')
+LIBERFADIR = os.path.join('liberfa', 'erfa')
+ERFA_SRC = os.path.join(LIBERFADIR, 'src')
 
 
 # https://mail.python.org/pipermail/distutils-sig/2007-September/008253.html
 class NumpyExtension(setuptools.Extension):
     """Extension type that adds the NumPy include directory to include_dirs."""
 
-    def __init__(self, *args, **kwargs):
```

Comment:
I removed this as we do not use cython so it seemed meaningless.

Reply:
+1